docs(login examples): Path Fixes for  login and firebase login examples (#3227)
felangel/bloc: docs/flutterfirebaselogintutorial.md

- [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), a Flutter widget which provides a bloc to its children.
- [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), a Flutter widget that handles building the widget in response to new states.

> - Using Cubit instead of Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)

Reviewer: ```suggestion
- Using Cubit and Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```

### Setup

> We'll start by creating `packages/authentication_repository` in the root project folder and a `pubspec.yaml` under it.

Reviewer: ```suggestion
We'll start by creating `packages/authentication_repository` and a `pubspec.yaml` at the root of the project.
```

[pubspec.yaml](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_firebase_login/packages/authentication_repository/pubspec.yaml ':include')

Next, we can install the dependencies by running:

```sh

> flutter pub get

Reviewer: ```suggestion
flutter packages get
```

> flutter pub get

Reviewer: flutter packages get should be fine as well.
